### PR TITLE
More rogue stuff

### PR DESCRIPTION
**Pull request by syrifgit**
dont ask why but the envenom at 5 cp bricks the whole spec. Will revist later.

- fix charged points
- final checks on Sin rogue pack
- properly return sub rogue covenant

### CODE REVIEW

**Comment by syrifgit** [comment]
Envenom option may have been a coincidence, but its not currently referenced/used so I'll leave it disabled as to not mislead people into thinking it does something.

File: TheWarWithin/RogueAssassination.lua [hunk 1]
```diff
@@ -2954,4 +2955,4 @@ spec:RegisterSetting( "allow_shadowmeld", nil, {
...
...21xY8RAFMezJ(FPhLyqZMSSBV2w4gYRLcg8UhGQWn4Y3WFn8dB21lMaCpIGwOINVllLcYBSOnij8TSQB6G602AaZAynZ2CyMQxbEBo)QiG1C884DAaJYMNqS8nV9cCTU(dg7ugbYQMsgTQIGrXIW3zBGzbKsXnwRQDFmttLXKeD5LfSsqS7Tvo3(A0Ug54D3Lxyo8WyYK5bYIeO6hTMEC5HInOn(2ENI4Vr2gnWY5GcnbqLuNmC4z1n(0Et6ZLr5AqINdznRZcznEuTTjIvRhEW2dC7FUpMpDw7(WvBWWW5T2dRBFCHmKnDJO)AET0GPmDcG9IiBtMYN9C3LCGNafvMcQVCXW0RkBbnSkNlWAdnhypSZNW850ltD(qm8OW8gldSgklBxJh2RVG2Qc0X4XtgkcxWm1XOG(aroHXw3RGvXwCBCXbDJr2XBioIC8ik618KuMyglz4KiNL4U22F0gLA2lvNpYEFbgqRY7bcXRuz1MvTnC81GDDsTObrTIcSlnTanIyaY1r0A6ZQnlwgwNwJnrwPttcGep)iREAGKPOGA8c)GXTWA6dE6tFkOami0UAEHv5SUQZwTFpsbuTzGKc8tg1mhtKfRO50iTNkuVZebNfThRGQQuTWwSfXStLuXXSv3xVeOhBGD1Rf7v3irYe9MQOLl)jXziqv)6Vtzjd3i734aP0BkeN0glz9l2kAUNMuX95COU6ZJGa3TRMDlDlrpntJI57C(QlKihmOr1J81uaJ9JyTGiUkfpIynGcVQvUFOkoohvlBYONpwDAJiUMjXnj13M06CpGbbYpURxcr6jfcSfRKsqVVPFq8A66z7vZk6F6zIip5lkATFo73yl8pcC75vSscZQeuZvrRPwz0mS8)xJn8sWYQOBFa0WtyPlRV1tXphKqtwwilu9zclrwJzrC4S6XmOb0wSgZIGgLrATsRN1v51VvTjk8QK8CGO8IJw3q6ordHuLtWY4)yTgTw8G2cX6ptZk3ah5NXtE5o1LiR3Qy(9NoyFjDXKPW6AEhJdjT7jLM3NCN)Szo(OOJ916oWwjGHRWKzbX3Xcsygi(NKEbedOjTufwcJQ2W7KUq0AG)Ue113guVPOmrFo8FczQxAC0iAmOAPuF4jHPqcfEzKAaXdeUc5nla6kt8NHy3k1p(seKNNEPJDr1LqM4e87RxaHU6NhF2Byxx0x9))d]] )
```
Comment (Hekili) [inline TheWarWithin/RogueAssassination.lua:2958]: Is there a .simc file update or is this a missed import from a previous .simc file update?

**Review by Hekili — CHANGES_REQUESTED** [review]
One question.

File: TheWarWithin/RogueAssassination.lua (hunk 1 above)
Comment (syrifgit) [reply TheWarWithin/RogueAssassination.lua:2958]: This should match the simc that's associated with this pr, so if there's no diff then the pack string was wrong previously

**Pull request merged**